nav_msgs/Odometry[] positions
float64[] yaws
int32[] labels
